nrf_security/Kconfig.legacy

```diff
@@ -254,6 +254,15 @@ config OBERON_ONLY_ENABLED
 
 menu "Legacy mbed TLS crypto APIs"
 
+config MBEDTLS_MPI_WINDOW_SIZE
+	int
```

Comment:
Do we wish for it to be user-configurable what the value is when CC3X_BACKEND is disabled? If so we should have a prompt that is conditional on CC3X_BACKEND.

If we never want this to be user-configurable we should make it clearer to the user that this is a constant by hardcoding 6 in the mbedtls header file.